src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, Arial, sans-serif;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 400;
+}
+
+html {
+  font-weight: 300;
+}
+
+body {
+  margin: 0;
+}
+
+.searchBar {
+  display: flex;
+  position: absolute;
+  align-items: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 100%;
+  margin-left: 48px;
+}
```

Comment:
You have a margin on all sides of your `.searchBar` class. It's recommended to be consistent with your margins. Add only top or bottom, but not both. This is to avoid potential margin collapse. You might want to reconsider this style.